internal/cli/atlas/deployments/options/deployment_opts.go

```diff
@@ -247,3 +245,50 @@ func (opts *DeploymentOpts) ValidateAndPromptDeploymentType() error {
 func (opts *DeploymentOpts) IsAtlasDeploymentType() bool {
 	return strings.EqualFold(opts.DeploymentType, AtlasCluster)
 }
+
+func (opts *DeploymentOpts) LocalDeploymentPreRun(ctx context.Context) error {
+	if !localDeploymentSupportedByOs() {
+		_, _ = log.Warningln("Local deployments are not supported on this OS, to see local deployments requirements visit https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/.")
```

Comment:
@mongodb/docs-cloud-team asking for review here

Reply:
LGTM